release 0.12.0-rc-1 (#615)
streamnative/function-mesh: CHANGELOG.md

# Function Mesh v0.12.0 Release Notes

## v0.12.0 What's New

* [Controller] Ensure the Pod Security Standard follows restricted specification ([#574](https://github.com/streamnative/function-mesh/pull/574))
* [Controller] Fix `fsGroup` not work when volume mount use an existing directory as subpath ([#577](https://github.com/streamnative/function-mesh/issues/577))
* [Controller] Add the default value for `minReplicas` and `replicas` ([#582](https://github.com/streamnative/function-mesh/pull/582))

> * [Controller] Introduce `ConnectorCatalog` CRD ([#585](https://github.com/streamnative/function-mesh/pull/585), [#610](https://github.com/streamnative/function-mesh/pull/610))

Reviewer: ```suggestion
* [Controller] Introduce the `ConnectorCatalog` CRD ([#585](https://github.com/streamnative/function-mesh/pull/585), [#610](https://github.com/streamnative/function-mesh/pull/610))
```

> * [Controller] Allow passing configs via ENV ([#587](https://github.com/streamnative/function-mesh/pull/587))

Reviewer: ```suggestion
* [Controller] Allow passing Function Mesh Operator configurations through the environment variables ([#587](https://github.com/streamnative/function-mesh/pull/587))
```

> * [Controller] Allow Function/Connector scale to zero when HPA is disabled ([#590](https://github.com/streamnative/function-mesh/pull/590))

Reviewer: ```suggestion
* [Controller] Allow scaling a function or connector to zero when HPA is disabled ([#590](https://github.com/streamnative/function-mesh/pull/590))
```

> * [Controller] Fix `SubscriptionType` is not handle properly ([#592](https://github.com/streamnative/function-mesh/pull/592))

Reviewer: ```suggestion
* [Controller] Fix the issue that `SubscriptionType` is not handled properly ([#592](https://github.com/streamnative/function-mesh/pull/592))
```

> * [Controller] Fix `DeadLetterTopic` is set to an incorrect value in some specific cases ([#595](https://github.com/streamnative/function-mesh/pull/595))

Reviewer: ```suggestion
* [Controller] Fix the issue that `DeadLetterTopic` is set to an incorrect value in some specific cases ([#595](https://github.com/streamnative/function-mesh/pull/595))
```

> * [Controller] Fix `MaxPendingAsyncRequests` is not working ([#599](https://github.com/streamnative/function-mesh/pull/599))

Reviewer: ```suggestion
* [Controller] Fix the issue that `MaxPendingAsyncRequests` is not working ([#599](https://github.com/streamnative/function-mesh/pull/599))
```

> * [Controller] Fix liveness probe is not working ([#604](https://github.com/streamnative/function-mesh/pull/604))

Reviewer: ```suggestion
* [Controller] Fix the issue that the liveness probe is not working ([#604](https://github.com/streamnative/function-mesh/pull/604))
```

> * [Controller] Add support to configure multiple PVs to CR ([#605](https://github.com/streamnative/function-mesh/pull/605))

Reviewer: ```suggestion
* [Controller] Support configuring multiple PVs to a CR ([#605](https://github.com/streamnative/function-mesh/pull/605))
```
